### PR TITLE
feature-OBSDEF-12024-svc_transfer

### PR DESCRIPTION
**Pull request by d1rksm33ts**
## Purpose
[OBSDEF-12024](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-12024)
_List the changes for this PR_

## PR checklist
- [ ] make test passed
- [ ] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
_Paste URL of charts-custom-ci job here_

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: supportassist/templates/supportassist-app-configmap.yaml [hunk 1]
```diff
@@ -99,3 +99,16 @@ data:
             - -folder
             - {{ .Values.telemetry.health.folder}}
           disabled: {{ default false .Values.telemetry.health.disabled }}
+        - name: data
```
Comment (vgupta-mickey) [inline supportassist/templates/supportassist-app-configmap.yaml:102]: I would say please use some more meaningful name for the telemetry report

File: supportassist/values.yaml [hunk 2]
```diff
@@ -160,6 +160,14 @@ telemetry:
     disabled: false
     folder: health
     transferType: CLOUDIQ
+  data:
+    image:
+      repository: telemetry-upload
+      tag: 2.83.0-237.21084d4 # rfw-update-this telemetry-upload-image
+    schedule: "*/30 * * * *"
+    disabled: false
+    folder: data
```
Comment (vgupta-mickey) [inline supportassist/values.yaml:169]: so I am assuming that some other cronjob will be creating a folder in the deck-support-store with the following path /telemetry/objectscale/OBJECTSCALE/data

**Review by vgupta-mickey — APPROVED** [review]

File: supportassist/templates/supportassist-app-configmap.yaml (hunk 1 above)
Comment (d1rksm33ts) [reply supportassist/templates/supportassist-app-configmap.yaml]: Using svc_transfer

File: supportassist/values.yaml (hunk 2 above)
Comment (d1rksm33ts) [reply supportassist/values.yaml:169]: Corrent

**Review by eddiepavkovic — APPROVED** [review]
ok

**Pull request merged**